python/hsfs/engine/spark.py

```diff
@@ -102,14 +102,14 @@ def set_job_group(self, group_id, description):
 
     def register_on_demand_temporary_table(self, on_demand_fg, alias, options={}):
```

Comment:
Why not following the same pattern as for Java?

Reply:
true, refactored to same pattern